Sources/MongoSwift/BSON/Document.swift

```diff
@@ -57,17 +61,30 @@ extension Document {
     }
 
     /**
-     * Initializes a `Document` from a pointer to a bson_t. Uses a copy
-     * of `bsonData`, so the caller is responsible for freeing the original
-     * memory.
+     * Initializes a `Document` from a pointer to a `bson_t` by making a copy of the
+     * data. The caller is responsible for freeing the original `bson_t`.
```

Comment:
I don't think this is always correct. For instance, in the APM code above we copy the pointer, and are _not_ responsible for freeing the original `bson_t`

Reply:
good point, I updated it to just say that whoever owns the `bson_t` is responsible for freeing it.